DNN Platform/Modules/ResourceManager/App_LocalResources/EditFolderMapping.ascx.resx

```diff
@@ -0,0 +1,168 @@
+                <xsd:element name="comment" type="xsd:string" minOccurs="0" msdata:Ordinal="2" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" msdata:Ordinal="1" />
+              <xsd:attribute name="type" type="xsd:string" msdata:Ordinal="3" />
+              <xsd:attribute name="mimetype" type="xsd:string" msdata:Ordinal="4" />
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="resheader">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" />
+            </xsd:complexType>
+          </xsd:element>
+        </xsd:choice>
+      </xsd:complexType>
+    </xsd:element>
+  </xsd:schema>
+  <resheader name="resmimetype">
+    <value>text/microsoft-resx</value>
+  </resheader>
+  <resheader name="version">
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="ControlTitle.Text" xml:space="preserve">
+    <value>{0} Folder Type</value>
+  </data>
+  <data name="ControlTitle_editfoldermapping.Text" xml:space="preserve">
+    <value>Edit Folder Mapping</value>
+  </data>
+  <data name="Description.Text" xml:space="preserve">
+    <value>Use this dialog to define a new folder type.</value>
+  </data>
+  <data name="DuplicateMappingName.Text" xml:space="preserve">
+    <value>A mapping with the same name already exists.  The changes have not been saved.</value>
+  </data>
+  <data name="FolderProviderSettings.Text" xml:space="preserve">
+    <value>Folder Provider Settings</value>
+  </data>
+  <data name="GeneralSettings.Text" xml:space="preserve">
+    <value>General Settings</value>
+  </data>
+  <data name="FolderProviderLabel.Help" xml:space="preserve">
+    <value>Select the folder provider to use.</value>
+  </data>
+  <data name="FolderProviderLabel.Text" xml:space="preserve">
+    <value>Folder Provider</value>
+  </data>
+  <data name="NameLabel.Help" xml:space="preserve">
+    <value>Enter the name of this folder mapping.</value>
+  </data>
+  <data name="NameLabel.Text" xml:space="preserve">
+    <value>Name</value>
+  </data>
+  <data name="RequiredFields.Text" xml:space="preserve">
+    <value>Indicates required fields.</value>
+  </data>
+  <data name="SelectFolderProvider.Text" xml:space="preserve">
+    <value>- Select a Folder Provider -</value>
```

Comment:
@valadas does this follow the pattern used elsewhere?

Reply:
It was copied directly from the old one... You have any suggestion here?

Reply:
No, I just didn't compare - I was being lazy I suppose.  😆